src/api/index.ts

```diff
@@ -8,7 +8,11 @@ import merge from 'deepmerge'
 
 const requestInterceptor = async (config: any) => {
   if (apiConfig.token) {
-    config.headers["Authorization"] =  "Bearer "+apiConfig.token;
+    if(apiConfig.systemType === "MOQUI") {
```

Comment:
This change is already handled in https://github.com/hotwax/oms-api/pull/115, we just need updating the oms-api version